config/main.py

```diff
@@ -27,6 +27,9 @@
 SYSLOG_IDENTIFIER = "config"
 VLAN_SUB_INTERFACE_SEPARATOR = '.'
 ASIC_CONF_FILENAME = 'asic.conf'
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+DEFAULT_CONFIG_DB_FILE = '/etc/sonic/config_db.json'
+NS_PREFIX = 'asic'
```

Comment:
Suggest to rename to `NAMESPACE_PREFIX` for readability

Reply:
Renamed to NAMESPACE_PREFIX